README.md

```diff
@@ -13,6 +121,107 @@ The Unbiased API server can be configured using environment variables. The follo
 - `UNBIASED_HOST`: The host on which the API will run. Defaults to `0.0.0.0` if not provided.
 - `UNBIASED_PORT`: The port on which the API will listen. Defaults to `3838` if not provided.
 
+# Getting started with **unbiased**
```

Comment:
Let's give examples from docker perspective (how to docker run with env variables)